apps/gro/controllers/address.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const BaseController = require('hof').controllers.base;
+const ErrorController = require('hof').controllers.error;
+const _ = require('lodash');
+
+module.exports = class Address extends BaseController {
+  constructor(options) {
+    super(options);
+  }
+
+  getValues(req, res, callback) {
+    res.locals.backLink = req.sessionModel.get('previous-step');
```

Comment:
This is because you are adding an extra step to process the api call. This isn't how the controller is intended to be used, and as such you have to try and override default behaviour such as persisting errors via the sessionModel and manually setting the back link.  I have a version of this with the logic via a model in a controller on the postcode step hooking into the `process` method. This avoids the need to add an extra empty step